### PR TITLE
Lograge formatter

### PR DESCRIPTION
**Pull request by luxflux**
Adds a formatter to print the log lines with help of [lograge](https://github.com/roidrage/lograge).

### CODE REVIEW

**Comment by aserafin** [comment]
Thank you @luxflux, looks good 👍 I will be merging this and releasing to rubygems today/tomorrow :)

**Comment by jcope2013** [comment]
will this be merged soon?

**Comment by aserafin** [comment]
@jcope2013 thanks for reminding me and @luxflux sorry for the delay :) This is now merged and brand new&shiny 1.6.0 version pushed to rubygems 🎉

File: lib/grape_logging/formatters/lograge.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    class Lograge
+      def call(severity, datetime, _, data)
+        time = data.delete :time
+        attributes = {
+          severity: severity,
+          duration: time[:total],
+          db: time[:db],
+          view: time[:view]
+        }.merge(data)
+        ::Lograge.formatter.call(attributes) + "\n"
```
Comment (stanhu) [inline lib/grape_logging/formatters/lograge.rb]: I noticed this omits `datetime`. Can we add that?

**Pull request merged**